api/models/templates/app.tmpl

```diff
@@ -633,15 +634,18 @@
     },
     "Volumes": [
       {{ range .MountableVolumes }}
-        "{{ . }}",
+        {{ if eq .Host "/var/run/docker.sock" }}
```

Comment:
Yes, this is the only special volume we support. Matt is working on the docs for this stuff.